electrum/gui/qml/android_res/layout/scanner_layout.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<FrameLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+        android:id="@+id/content_frame"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <TextView
+        android:id="@+id/hint"
+        android:layout_gravity="center|top"
+        android:text="Scan a QR code."
+        android:layout_width="wrap_content"
+        android:textColor="#ffffff"
+        android:layout_height="wrap_content" />
```

Comment:
I made this look nicer in https://github.com/sombernight/electrum/commit/638f2d885cd9fb4dae47012d59dc0dbad420f838

Compare:
<details>
 <summary>before</summary>

![Screenshot_20231031_190414_Electrum](https://github.com/spesmilo/electrum/assets/29142493/6748cea1-3bb6-4415-96a5-b82511e359e3)
</details>

<details>
 <summary>after</summary>

![Screenshot_20231031_190948_Electrum](https://github.com/spesmilo/electrum/assets/29142493/bbd2c1e3-d7df-404c-a5f9-2374d458cb53)
</details>

Reply:
Looks much nicer indeed. Feel free to just add to this branch directly